homework011/xorm-student.go

```diff
@@ -10,10 +10,36 @@ import (
 // 問題6
 
 // Student構造体を実装してください。
+type Student struct {
+	ID    int    `xorm:"id"`
+	Name  string `xorm:"name"`
+	Class int    `xorm:"class"`
+	Age   int    `xorm:"age"`
+	Score int    `xorm:"score"`
+}
 
 // insertSumStudent を実装してください。
 // 中身を適宜変更してください。引数も変更して大丈夫です。
 func insertSumStudent(engine xorm.Engine, students []Student) {
+	var i int
+	var sum = 0
+	for i = 0; i <= 2; i++ {
+		student := Student{
+			ID:    students[i].ID,
+			Name:  students[i].Name,
+			Class: students[i].Class,
+			Age:   students[i].Age,
+			Score: students[i].Score,
+		}
```

Comment:
```suggestion
	for _, student := range students {
	
```

と書きましょう。
i <= 2　にしていると、studentが増えたら破綻してしまします。
また、少し助長です。